identifiers/country-my/federal_states_terriroties.csv

```diff
@@ -0,0 +1,17 @@
+id,name
+ocd-division/country:my/state:01,Johor
+ocd-division/country:my/state:02,Kedah
+ocd-division/country:my/state:03,Kelantan
+ocd-division/country:my/state:04,Melaka
+ocd-division/country:my/state:05,Negeri Sembilan
+ocd-division/country:my/state:06,Pahang
+ocd-division/country:my/state:07,Pulau Pinang
+ocd-division/country:my/state:08,Perak
+ocd-division/country:my/state:09,Perlis
+ocd-division/country:my/state:12,Sabah
+ocd-division/country:my/state:13,Sarawak
+ocd-division/country:my/state:10,Selangor
+ocd-division/country:my/state:11,Terengganu
+ocd-division/country:my/federal_territory:14,Wilayah Persekutuan Kuala Lumpur

```

Comment:
What do you think about using `territory` instead of `federal_territory` for some consistency with other country OCD IDs?

Reply:
SGTM. Modified.